Add Initiatives to Admin manual

andreslucena opened the pull request:
Add Initiatives page in the admin manual

paulinebessoles commented on en/modules/admin/pages/initiatives.adoc:
```diff
@@ -1,14 +1,369 @@
+
+|Enable attachments
+|Optional
+|Wheter it'll allow to upload attachments to initiatives.
```
```suggestion
|Whether it'll allow to upload attachments to initiatives.
```

paulinebessoles commented on en/modules/admin/pages/initiatives.adoc:
```diff
@@ -1,14 +1,369 @@
+
+|Enable promoting committee
+|Optional
+|Wheter if it will allow individual initiatives (made by only one person) or it will always have to have a collective (multiple promoters, being multiple individuals or a user group).
```
```suggestion
|Whether it will only allow individual initiatives (made by only one person) or it will always have to have a collective (multiple promoters, being multiple individuals or a user group).
```

paulinebessoles commented on en/modules/admin/pages/initiatives.adoc:
```diff
@@ -1,14 +1,369 @@
+
+image:initiative_commitee_members_send_to_technical_false.png[Initiative promotor committee: edit screen]
+image:initiative_edit_commitee_members.png[Initiative promotor committee: edit screen]
+image:initiative_commitee_members_sent.png[Initiative promotor committee: edit screen]
```
```suggestion
image:initiative_commitee_members_ask.png[Initiative promotor committee: edit screen]

The people invited with the link will be able to read the initiative and ask to be part of the committee.

```

paulinebessoles commented on en/modules/admin/pages/initiatives.adoc:
```diff
@@ -1,14 +1,369 @@
+image:initiative_commitee_members_send_to_technical_false.png[Initiative promotor committee: edit screen]
+image:initiative_edit_commitee_members.png[Initiative promotor committee: edit screen]
+image:initiative_commitee_members_sent.png[Initiative promotor committee: edit screen]
+image:initiative_commitee_members_ask.png[Initiative promotor committee: edit screen]
```
```suggestion
image:initiative_commitee_members_sent.png[Initiative promotor committee: edit screen]

The request is sent and the author can then and accept or refuse it in the initiative edit form. 

```

paulinebessoles commented on en/modules/admin/pages/initiatives.adoc:
```diff
@@ -1,14 +1,369 @@
+each request, by opening the Admin Dashboard link in the user menu, editing their
+initiative, clicking "Committee members" and then approving each member.
+
+image:initiative_commitee_members_send_to_technical_false.png[Initiative promotor committee: edit screen]
```
```suggestion
image:initiative_commitee_members_send_to_technical_false.png[Initiative promotor committee: edit screen]

The initiative cannot be sent to technical validation until it meets the minimum number of committee members required. You can find the invitation link in the edit form.

```

paulinebessoles commented on en/modules/admin/pages/initiatives.adoc:
```diff
@@ -1,14 +1,369 @@
+initiative, clicking "Committee members" and then approving each member.
+
+image:initiative_commitee_members_send_to_technical_false.png[Initiative promotor committee: edit screen]
+image:initiative_edit_commitee_members.png[Initiative promotor committee: edit screen]
```
```suggestion
image:initiative_edit_commitee_members.png[Initiative promotor committee: edit screen]

Share the invitation link with the people you'd like to have on your committee. 

```

paulinebessoles commented on en/modules/admin/pages/initiatives.adoc:
```diff
@@ -1,14 +1,369 @@
+image:initiative_edit_commitee_members.png[Initiative promotor committee: edit screen]
+image:initiative_commitee_members_sent.png[Initiative promotor committee: edit screen]
+image:initiative_commitee_members_ask.png[Initiative promotor committee: edit screen]
+image:initiative_commitee_members_approve.png[Initiative promotor committee: edit screen]
```
```suggestion
image:initiative_commitee_members_approve.png[Initiative promotor committee: edit screen]

```

paulinebessoles reviewed (commented):
The documentation and the diagrams are really clear, congrats for the improvements! 
I changed the order of 2 images and added some captions on the promotor committee section, if you want to take a look at it :)

andreslucena commented:
> The documentation and the diagrams are really clear, congrats for the improvements! 

:blush: 

> I changed the order of 2 images and added some captions on the promotor committee section, if you want to take a look at it :)

I accepted the changes, it's much clearer, thanks!
